### PR TITLE
Koa 2.x support

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by ofrobots** [comment]
Note parallel dependency on #463. Whichever lands last needs to some more work?

File: src/plugins/plugin-koa.js [hunk 1]
```diff
@@ -18,83 +18,111 @@
...
+    return next();
   };
 }
 
 module.exports = [
   {
     file: '',
-    versions: SUPPORTED_VERSIONS,
+    versions: '1.x',
+    patch: function(koa, api) {
```
Comment (ofrobots) [inline src/plugins/plugin-koa.js]: nit: Dedupe this function.

Comment (matthewloring) [inline src/plugins/plugin-koa.js]: done.

**Comment by matthewloring** [comment]
CLA should be fine given that the included commit was already opened in https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/442 which received CLA approval.

**Pull request merged**